### PR TITLE
Correct and Update Chinese Simplified translation

### PR DESCRIPTION
**Pull request by OrO-c**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -70,7 +70,7 @@
     <string name="model_icon">模型的图标</string>
     <string name="send">发送</string>
     <string name="copy_text">复制</string>
-    <string name="retry">重新生成</string>
+    <string name="retry">重试</string>
```
Comment (Taewan-P) [inline app/src/main/res/values-zh-rCN/strings.xml]: Hi,

From the previous PR, 0e59ebe6194a0a7c4be01779b27ab63149aea80c to 0b433768c3c94be5edb0ba973658216ff5055b57, the string "retry" is keep changing. Which one is correct? `重新生成` or `重试`?

Comment (OrO-c) [reply app/src/main/res/values-zh-rCN/strings.xml]: 重新生成

**Review by Taewan-P — APPROVED** [review]
Thanks for the contribution!

**Comment by Taewan-P** [comment]
@OrO-c Today, I just got approval from [Hosted Weblate](https://hosted.weblate.org/engage/gptmobile/) for libre open source project. For future translation updates, it would be more easier to add them at Weblate. Thanks!

**Comment by OrO-c** [comment]
> Today, I just got approval from Hosted Weblate for libre open source project. For future translation updates, it would be more easier to add them at Weblate. Thanks!

ok

**Pull request merged**